CHANGELOG.next.asciidoc

```diff
@@ -17,6 +17,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Drop support for Debian 10 and upgrade statically linked glibc from 2.28 to 2.31 {pull}41402[41402]
 - Fix metrics not being ingested, due to "Limit of total fields [10000] has been exceeded while adding new fields [...]". The total fields limit has been increased to 12500. No significant performance impact on Elasticsearch is anticipated. {pull}41640[41640]
 - Set default kafka version to 2.1.0 in kafka output and filebeat. {pull}41662[41662]
+- Add regex pattern matching to add_kubernetes_metadata processor {pull}41903[41903]
```

Comment:
shouldn't it be added in `Added`, instead of `Breaking changes`?

Reply:
My mistake, good catch! Fixed now.